<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPhraseEditor</class>
 <widget class="QWidget" name="CustomPhraseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>356</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="tableView_"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_">
     <item>
      <widget class="QPushButton" name="addButton_">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="batchEditButton_">
       <property name="text">
        <string>Batch &amp;Edit</string>
       </property>
       <property name="icon">
        <iconset theme="document-edit"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton_">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton_">
       <property name="text">
        <string>&amp;Clear</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
